homework/01LessonOne.js

```diff
@@ -107,34 +107,33 @@ var notNotTrue;
 //
 // Your Answer Goes Here:
 //
-// ****
-
+// **** When you use &&  you want know if your conditional results are true or false. When you use || you want all your conditonal results to be true.
```

Comment:
Whoops! This is a little off. Let's talk about it! And read here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators